controller/array_action/array_mediator_xiv.py

```diff
@@ -305,26 +302,26 @@ def get_object_by_id(self, object_id, object_type):
             return self._generate_snapshot_response(cli_object)
         return self._generate_volume_response(cli_object)
 
-    def create_snapshot(self, name, volume_name, pool_id=None):
-        logger.info("creating snapshot {0} from volume {1}".format(name, volume_name))
-
+    def create_snapshot(self, volume_id, snapshot_name, pool=None):
```

Comment:
@zingero do you think we should continue to ignore the snapshotclass pool with xiv? or should we add some error?

Reply:
@oriyarde
If the user writes the source pool then we should let it go. Otherwise, we should fail the request and add a descriptive message like:
`Can create SC in the source pool only`.
Also, we should edit the documentation accordingly.

Reply:
done